charts/prefect-agent/README.md

```diff
@@ -28,13 +28,13 @@ Prefect Agent application bundle
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | agent.affinity | object | `{}` | affinity for agent pods assignment |
-| agent.config.accountId | string | `""` | prefect account ID |
-| agent.config.apiKeySecret.enabled | bool | `true` | attach prefect-api-key secret to agent container |
-| agent.config.apiKeySecret.key | string | `"key"` | prefect API secret key |
-| agent.config.apiKeySecret.name | string | `"prefect-api-key"` | prefect API secret name |
-| agent.config.apiUrl | string | `"https://api.prefect.cloud"` | prefect API url |
-| agent.config.workQueueName | string | `""` | prefect workqueue name |
-| agent.config.workspaceId | string | `""` | prefect workspace ID |
+| agent.apiConfig | string | `"cloud"` | one of 'cloud' or 'orion' |
+| agent.cloudApiConfig.accountId | string | `""` | prefect account ID |
+| agent.cloudApiConfig.apiKeySecret.key | string | `"key"` | prefect API secret key |
+| agent.cloudApiConfig.apiKeySecret.name | string | `"prefect-api-key"` | prefect API secret name |
+| agent.cloudApiConfig.cloudUrl | string | `"https://api.prefect.cloud/api"` | prefect cloud API url; the full URL is constructed as https://cloudUrl/accounts/accountId/workspaces/workspaceId |
+| agent.cloudApiConfig.workspaceId | string | `""` | prefect workspace ID |
+| agent.config.workQueueName | string | `"default"` | prefect workqueue name |
```

Comment:
if we're going to make breaking changes now, then I'd propose that we change this type to an array of strings (for multi work queue support) and also just call this `workQueue` or `workQueues`, because users can specify either a name or UUID here

something like:

```yaml
agent:
  config:
    workQueues:
      - abc
      - def
```

I think there's template functions that can do an array join for us

Reply:
good call - done!